libkbfs/kbfs_ops_test.go

```diff
@@ -706,6 +706,7 @@ func TestKBFSOpsGetBaseDirChildrenCacheSuccess(t *testing.T) {
 	ops := getOps(config, id)
 	n := nodeFromPath(t, ops, p)
 
+	config.mockMdserv.EXPECT().FastForwardBackoff().AnyTimes()
```

Comment:
You can put this in `kbfsOpsInit()` in you want, so we don't have to call it in every test.